lua/packer/plugin_types/git.lua

```diff
@@ -454,6 +491,30 @@ git.setup = function(plugin)
     end)()
     return r
   end
+
+  ---Reset the plugin to `plugin.commit`
+  plugin.revert = function ()
```

Comment:
Maybe this should be called something more like `revert_to` or similar, to avoid confusion with the `revert_last` method?